pkg/test/ginkgo/options_monitor_events_test.go

```diff
@@ -0,0 +1,183 @@
+							Message: "reason/NetworkNotReady network is wacked: container runtime network is DIFFERENT. Has your DIFFERENT network provider started? (8 times)",
+						},
+						From: from.Add(-90 * time.Second),
+						To:   to.Add(-89 * time.Second),
+					},
+				},
+			},
+		},
+		{
+			name: "locatorDifferent, msgMatch",
+			size: 1,
+			args: args{
+				events: []monitorapi.EventInterval{
+					{
+						Condition: monitorapi.Condition{
+							Locator: "node/ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0 hmsg/f33a7e39ac",
+							Message: "pathological/true reason/ErrorReconcilingNode roles/control-plane,master [k8s.ovn.org/node-chassis-id annotation not found for node ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0, macAddress annotation not found for node \"ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0\" , k8s.ovn.org/l3-gateway-config annotation not found for node \"ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0\"] (21 times)",
+						},
+						From: from.Add(-100 * time.Second),
+						To:   to.Add(-95 * time.Second),
+					},
+					{
+						Condition: monitorapi.Condition{
+							Locator: "node/ci-op-i20psv8m-6a467-xftbs-master-j6mzw-DIFFERENT",
+							Message: "reason/ErrorReconcilingNode roles/control-plane,master [k8s.ovn.org/node-chassis-id annotation not found for node ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0, macAddress annotation not found for node \"ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0\" , k8s.ovn.org/l3-gateway-config annotation not found for node \"ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0\"] (19 times)",
+						},
+						From: from.Add(-110 * time.Second),
+						To:   to.Add(-105 * time.Second),
+					},
+				},
+			},
+		},
+		{
+			name: "no patho events",
+			size: 0,
+			args: args{
+				events: []monitorapi.EventInterval{
+					{
+						Condition: monitorapi.Condition{
+							Locator: "node/ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0",
+							Message: "reason/ErrorReconcilingNode roles/control-plane,master [k8s.ovn.org/node-chassis-id annotation not found for node ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0, macAddress annotation not found for node \"ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0\" , k8s.ovn.org/l3-gateway-config annotation not found for node \"ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0\"] (5 times)",
+						},
+						From: from.Add(-100 * time.Second),
+						To:   to.Add(-95 * time.Second),
+					},
+					{
+						Condition: monitorapi.Condition{
+							Locator: "node/ci-op-i20psv8m-6a467-xftbs-master-j6mzw-1",
+							Message: "reason/ErrorReconcilingNode roles/control-plane,master [k8s.ovn.org/node-chassis-id annotation not found for node ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0, macAddress annotation not found for node \"ci-op-i20psv8m-6a467-xftbs-master-j6mzw-1\" , k8s.ovn.org/l3-gateway-config annotation not found for node \"ci-op-i20psv8m-6a467-xftbs-master-j6mzw-0\"] (4 times)",
+						},
+						From: from.Add(-110 * time.Second),
+						To:   to.Add(-105 * time.Second),
+					},
+				},
```

Comment:
Best approach here  might be to fully define your expected output events and compare to make sure the lists are the same. Add some testing with mixed messages and locators. This will help ensure we're not going to mistakenly pass.

Likely can do that with assert.ElementsEqual or reflect.DeepEqual.